addons/calendar/models/calendar_alarm.py

```diff
@@ -72,3 +76,14 @@ def unlink(self):
         result = super(Alarm, self).unlink()
         self._update_cron()
         return result
+
+    def action_test(self):
+        ctx = dict(self.env.context, default_alarm_id=self.id)
+        return {
+            'name': _('Test Reminder'),
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'res_model': 'calendar.alarm.test',
+            'target': 'new',
+            'context': ctx,
```

Comment:
Doesn't work with just:
`'context': {'default_alarm_id': self.id}`
?

The rest of the context should be automatically propagated right?
(I know this is copied from "mailing.mailing.test", but I'm curious).

Reply:
I can do that but when the user will create a reminder from a calendar event, then when he will arrive in the modal, the event will not be pre-selected